### DISCUSSION TITLE
NotificationBar Component only show once

### DISCUSSION THREAD

**Discussion by bany21** [balajidharma/laravel-vue-admin-panel#27] [General]
Hello,

Thanks for the beautiful project. I used it to learn Laravel and made me easy to understand how laravel works.
I have an issue when trying to use NotificationBar Component. This component only shows/works once until I refresh the page.
I use this to show flash message. I see that NotificationBar element will show when flash message contains value by this code  v-if="$page.props.flash.message". I noticed that the NotificationBar has Dismiss Button that will make NotificationBar disappear. Inside that file, there is code v-if="!isDismissed". I think this code made the notification bar not show for the second or next request due to the Notification set to dismiss with a button that also dismissed with the NotificationBar itself. And there is no instruction to reset the isDismissed to default value or set flash message to null again.

**Comment by balajidharma** [comment]
I have created a new issue - https://github.com/balajidharma/laravel-vue-admin-panel/issues/28
